libvuln/migrations/06-delete-debian-update_operation.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM update_operation WHERE updater IN (
+	'debian-bullseye-updater', 'debian-buster-updater',
+	'debian-jessie-updater', 'debian-stretch-updater',
+	'debian-wheezy-updater');
```

Comment:
Happy to do this by changing the fingerprints like with `pyupio` but this way the GC should delete all the "old style" debian vulns on the initial update (after upgrade).